libcontainer/cgroups/fs2/create.go

```diff
@@ -95,13 +96,25 @@ func CreateCgroupPath(path string, c *configs.Cgroup) (Err error) {
 					}
 				}()
 			}
+			if rootless {
+				// Write cgroup.type explicitly.
+				// Otherwise ENOTSUP may happen.
```

Comment:
"may happen" where? When enabling controllers? Creating children cgroups?

Why the issue this code is trying to solve is rootless-only?

Reply:
Updated. This seems needed when running the test in  acontainer (`make integration`), regardless to rootless or rootful.